ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -653,4 +653,6 @@ SENTRY_API_KEY = '{{ localsettings.SENTRY_API_KEY }}'
 
 {% if localsettings.get('GREENHOUSE_API_KEY') %}
 GREENHOUSE_API_KEY = '{{ localsettings.GREENHOUSE_API_KEY }}'
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+LOCAL_LOGGING_LOGGERS = {{ localsettings.get('LOCAL_LOGGING_LOGGERS', {}) }}
```

Comment:
This will override https://github.com/dimagi/commcare-hq/pull/15880/files#diff-bdf3ecebd8379ca98cc89e545fc90899R1124